src/components/TaskBoard/index.jsx

```diff
@@ -1,29 +1,56 @@
-import React from "react";
-import { useSelector } from "react-redux";
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import AddTaskCard from "components/TaskCard/AddTaskCard.jsx";
 import TaskCard from "components/TaskCard";
 import style from "components/TaskBoard/index.module.scss";
 import EditTaskCard from "components/TaskCard/EditTaskCard";
+import Pagination from "components/Pagination";
+import { PAGINATION } from "utils/constant/form";
+import { paginationUpdate } from "store/actions/";
 const TaskBoard = () => {
   const createButtonState = useSelector((state) => state.createButton);
+  const pagination = useSelector((state) => state.paginationLength);
   const tasks = useSelector((state) => state.todo);
   const editableTask = useSelector((state) => state.editButton);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (createButtonState) {
+      dispatch(paginationUpdate(pagination - 1));
+    } else {
+      if (pagination < PAGINATION) {
+        dispatch(paginationUpdate(pagination + 1));
+      }
+    }
+  }, [createButtonState]);
 
   return (
-    <div className={style.taskBoard}>
-      {createButtonState && <AddTaskCard />}
-      {tasks.map((todo) =>
-        todo.id == editableTask ? (
-          <EditTaskCard key={todo.id} id={todo.id} task={todo.task} />
-        ) : (
-          <TaskCard
-            key={todo.id}
-            id={todo.id}
-            task={todo.task}
-            createdTime={todo.createdTime}
-            completed={todo.completed}
-          />
-        )
+    <div>
+      <div className={style.taskBoard}>
+        {createButtonState && <AddTaskCard />}
+        {tasks
+          .slice(0, pagination)
+          .map((todo) =>
+            todo.id == editableTask ? (
+              <EditTaskCard key={todo.id} id={todo.id} task={todo.task} />
+            ) : (
+              <TaskCard
+                key={todo.id}
+                id={todo.id}
+                task={todo.task}
+                createdTime={todo.createdTime}
+                completed={todo.completed}
+              />
+            )
+          )}
+      </div>
+
+      {tasks.length + createButtonState > PAGINATION ? (
```

Comment:
Could we be more explicit while naming variables ?
```suggestion
      {tasks.length + createButtonState > PAGINATION_LIMIT/PER_PAGE ? (
```